src/HeckeTypes.jl

```diff
@@ -2241,11 +2246,11 @@ mutable struct qAdicRootCtx
     lf = Hecke.factor_mod_pk(Array, H, 1)
     if splitting_field
       d = lcm([degree(y[1]) for y = lf])
-      R = qadic_field(p, d, precision = 1)[1]
+      R = QadicField(p, d, 1)[1]
```

Comment:
What is this change about? I thought QadicField is deprecated

Reply:
Huh, that should not be here. Looks like by accident some old file content snuck in. Will fix it ASAP